src/Game.java

```diff
@@ -0,0 +1,149 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.*;
+
+public class Game {
+    private static final Random RANDOM = new Random();
+
+    private final static int PLAY = 1;
+    private final static int EXIT = 2;
+
+    Scanner scanner = new Scanner(System.in);
+
+    private static final int MAX_ERRORS = 8;
+    private String wordToFind;
+    private char[] wordFound;
+    private int nbErrors;
+    private ArrayList<String> letters = new ArrayList<>();
+    private ArrayList<String> lettersFromFile = new ArrayList<>();
+
+    public void hangman(){
+    boolean isException = true;
+        while (isException) {
+        try {
+            int option = -1;
+            while (option != EXIT) {
+
+                System.out.println("Wybierz opcję:");
+                System.out.println("1 - graj");
+                System.out.println("2 - koniec programu");
+                option = scanner.nextInt();
+                scanner.nextLine();
+
+                switch (option) {
+                    case EXIT:
+                        System.out.println("Bye bye!");
+                        scanner.close();
+                        break;
+                    case PLAY:
+                        play();
+                        break;
+                    default:
+                        System.out.println("Wybrałeś nieprawidłową opcję");
+                }
+            }
+            isException = false;
+        } catch (InputMismatchException e) {
+            System.out.println("Podałeś niepoprawny znak.");
+            scanner.nextLine();
+        }
+    }
+}
+
+    private String nextWordToFind() {
+        return Keywords.WORDS[RANDOM.nextInt(Keywords.WORDS.length)];
+    }
+
+    private void addWord(String word) {
+        lettersFromFile.add(word);
+    }
+
+    private String wordToFindFromFile() {
+        String fileName = "words.txt";
+        try (
+                var fileReader = new FileReader(fileName);
+                var bufferedReader = new BufferedReader(fileReader);
+        ) {
+            while (bufferedReader.readLine() != null) {
+                addWord(bufferedReader.readLine());
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return lettersFromFile.get(RANDOM.nextInt(lettersFromFile.size())).toUpperCase();
+    }
+
+    private void newGame() {
+        nbErrors = 0;
+        letters.clear();
+        wordToFind = nextWordToFind();
+        wordFound = new char[wordToFind.length()];
+
+        Arrays.fill(wordFound, '*');
+    }
+
+    private boolean wordFound() {
+        return wordToFind.contentEquals(new String(wordFound));
+    }
+
+    private String wordFoundContent() {
```

Comment:
what is this method doing?

Reply:
The method returns the searched words found by the user in the String format. For better readability, we add spaces between each letter. But all in all this method is not really needed.

Reply:
So does it return words or letters?

Reply:
Why do you think adding spaces increases readability? Won't it confuse user?

Reply:
bump

Reply:
I thought so, that it looks better like "*" or later the exposed letters are spaced, for me it is more readable

Reply:
k